### PR TITLE
Optional single node AWS e2e for console-operator master presubmit

### PR DESCRIPTION
**Pull request by omertuc**

### CODE REVIEW

**Comment by omertuc** [comment]
@jhadvig

File: ci-operator/config/openshift/console-operator/openshift-console-operator-master.yaml [hunk 1]
```diff
@@ -89,6 +89,10 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-single-node
```
Comment (spadgett) [inline ci-operator/config/openshift/console-operator/openshift-console-operator-master.yaml:92]: Should this go in openshift-console-operator-release-4.8.yaml and openshift-console-operator-release-4.9.yaml, too?

**Review by spadgett — COMMENTED** [review]
Thanks @omertuc

File: ci-operator/config/openshift/console-operator/openshift-console-operator-master.yaml (hunk 1 above)
Comment (omertuc) [reply ci-operator/config/openshift/console-operator/openshift-console-operator-master.yaml:92]: I think this happens automatically after a merge. I'll verify and get back to you

Comment (omertuc) [reply ci-operator/config/openshift/console-operator/openshift-console-operator-master.yaml:92]: It seems to have happened in other repositories where we added this job. Let's try to merge - if it doesn't happen we'll add it in a separate PR

**Review by spadgett — APPROVED** [review]
/approve

**Pull request merged**